tsconfig.json

```diff
@@ -12,6 +12,7 @@
     "resolveJsonModule": true,
     "strict": true,
     "noUncheckedIndexedAccess": true,
+    "skipLibCheck": true,
```

Comment:
This should always be true, if there's an issue with the types of a library in `node_modules`, we don't want it to stop us from building.